routes/cartRouter.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const cartController = require("../controllers/cartController");
+const loginRequired = require("../utils/auth");
+
+const router = express.Router();
+
+router.get("", loginRequired, cartController.getCartList);
+router.post("/addCartItem", loginRequired, cartController.addCartItems);
```

Comment:
```suggestion
router.post("", loginRequired, cartController.createCart);
```
REST 아키텍쳐로 개발할 때에는 엔드포인트에 동사가 들어가지 않습니다.

Reply:
수정했습니다 !